Purchases/Caching/RCDeviceCache.h

```diff
@@ -37,6 +37,8 @@ NS_ASSUME_NONNULL_BEGIN
 
 - (void)clearPurchaserInfoCacheTimestamp;
```

Comment:
This could be private/protected now

Reply:
good catch. I hadn't removed it initially because it was used in tests, I'll move it to protected

Reply:
actually, it's still used in https://github.com/RevenueCat/purchases-ios/blob/fix/cached_version_returned_after_invalidating_purchaserinfo_cache/Purchases/Public/RCPurchases.m#L858

and in that instance, we don't want to delete cache in case the request fails